lang/po/new/sv.po

```diff
@@ -1,1772 +1,1758 @@
+#: mmu.cpp:1587
+msgid "MMU load failed"
+msgstr "MMU-laddning felade"
+
+# MSG_MMU_LOAD_FAILS c=15
+#: messages.c:70
+msgid "MMU load fails"
+msgstr "MMU-laddn felar"
+
+# MSG_MMU_OK_RESUMING c=20 r=4
+#: mmu.cpp:776
+msgid "MMU OK. Resuming..."
+msgstr "MMU OK. Aterupptar..."
+
+# MSG_MODE c=6
+#: messages.c:103
+msgid "Mode"
+msgstr "MMU Mode"
+
+# MSG_MK3_FIRMWARE_ON_MK3S c=20 r=4
+#: Marlin_main.cpp:882
+msgid "MK3 firmware detected on MK3S printer"
+msgstr "MK3-firmware uppt\xe1ckt pa MK3S-skrivare"
+
+# MSG_NORMAL c=7
+#: messages.c:107
+msgid "Normal"
+msgstr "Normal"
+
+# MSG_SILENT c=7
+#: messages.c:106
+msgid "Silent"
+msgstr "Tyst"
+
+# MSG_MMU_USER_ATTENTION c=20 r=3
+#: mmu.cpp:722
+msgid "MMU needs user attention."
+msgstr "MMU fordrar anv\xe1ndarens uppm\xe1rksamhet.."
+
+# MSG_MMU_POWER_FAILS c=15
+#: ultralcd.cpp:1517
+msgid "MMU power fails"
+msgstr "MMU str\xefmavbr"
+
+# MSG_STEALTH c=7
+#: messages.c:108
+msgid "Stealth"
+msgstr "Tyst"
+
+# MSG_AUTO_POWER c=10
+#: messages.c:105
+msgid "Auto power"
+msgstr "Auto kraft"
+
+# MSG_HIGH_POWER c=10
+#: messages.c:104
+msgid "High power"
+msgstr "H\xefg kraft"
+
+# MSG_MMU_CONNECTED c=18
+#: ultralcd.cpp:1997
+msgid "MMU2 connected"
+msgstr "MMU2 ansluten"
+
+# MSG_SELFTEST_MOTOR c=18
+#: messages.c:96
+msgid "Motor"
+msgstr "Motor"
+
+# MSG_MOVE_AXIS c=18
+#: ultralcd.cpp:5677
+msgid "Move axis"
+msgstr "Flytta axlar"
+
+# MSG_MOVE_X c=18
+#: ultralcd.cpp:4233
+msgid "Move X"
+msgstr "Flytta X"
+
+# MSG_MOVE_Y c=18
+#: ultralcd.cpp:4234
+msgid "Move Y"
+msgstr "Flytta Y"
+
+# MSG_MOVE_Z c=18
+#: ultralcd.cpp:4235
+msgid "Move Z"
+msgstr "Flytta Z"
+
+# MSG_NO_MOVE c=20
+#: Marlin_main.cpp:5852
+msgid "No move."
+msgstr "Ingen r\xefrelse."
+
+# MSG_NO_CARD c=18
+#: ultralcd.cpp:6697
+msgid "No SD card"
+msgstr "Inget SD-kort"
+
+# MSG_NA c=3
+#: messages.c:127
+msgid "N/A"
+msgstr "N/A"
+
+# MSG_NO c=4
+#: messages.c:71
+msgid "No"
+msgstr "Nej"
+
+# MSG_SELFTEST_NOTCONNECTED c=20
+#: ultralcd.cpp:8128
+msgid "Not connected"
+msgstr "Inte ansluten"
+
+# MSG_NEW_FIRMWARE_AVAILABLE c=20 r=2
+#: util.cpp:195
+msgid "New firmware version available:"
+msgstr "Ny firmware version tillg\xe1nglig:"
+
+# MSG_SELFTEST_FAN_NO c=19
+#: messages.c:92
+msgid "Not spinning"
+msgstr "Roterar inte"
+
+# MSG_WIZARD_V2_CAL c=20 r=8
+#: ultralcd.cpp:4838
+msgid "Now I will calibrate distance between tip of the nozzle and heatbed surface."
+msgstr "Nu ska jag kalibrera avstandet mellan munstyckets spets och v\xe1rmeb\xe1ddsytan."
+
+# MSG_WIZARD_WILL_PREHEAT c=20 r=4
+#: ultralcd.cpp:4967
+msgid "Now I will preheat nozzle for PLA."
+msgstr "Nu ska jag f\xefrv\xe1rma munstycket f\xefr PLA."
+
+# MSG_NOZZLE c=12
+#: messages.c:72
+msgid "Nozzle"
+msgstr "Munstycke"
+
+# MSG_DEFAULT_SETTINGS_LOADED c=20 r=6
+#: Marlin_main.cpp:1605
+msgid "Old settings found. Default PID, Esteps etc. will be set."
+msgstr "Gamla inst\xe1llningar hittades. Standard PID, Esteps etc. kommer att st\xe1llas in."
```

Comment:
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.